.github/workflows/push-helm-charts.yaml

```diff
@@ -45,6 +45,12 @@ on:
         required: false
         default: false
         type: boolean
+      checkout-ref:
+        description:
+          Specify a branch to checkout
```

Comment:
This could also be other types of refs like a tag or sha.  Seems reasonable to either copy or link to the docs from `actions/checkout` which completely describe this behavior.

```
    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
```